### PR TITLE
i18n(fr): Updated `getting-started.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content
- Changes to Starlight code
- Something else!

#### Description

- What does this PR change? Give us a brief description.
Updated `getting-started.mdx` based on this commit 35ec9e2ff42150231979217414618528cc72a93a

### CODE REVIEW

File: docs/src/content/docs/fr/getting-started.mdx [hunk 1]
```diff
@@ -85,3 +85,13 @@ Si vous oubliez quelque chose d'important, Starlight vous le fera savoir.
 Une fois que vous avez créé et personnalisé votre site Web Starlight, vous pouvez le déployer sur un serveur Web ou une plateforme d'hébergement de votre choix, y compris Netlify, Vercel, GitHub Pages et bien d'autres.
 
 [Pour en savoir plus sur le déploiement d'un site Astro, consultez la documentation Astro](https://docs.astro.build/fr/guides/deploy/)
+
+## Dépannage de Starlight
+
+Les informations de configuration du projet Starlight [configuration du projet](/fr/reference/configuration/) et [configuration du frontmatter de la page individuelle](/fr/reference/frontmatter/) sont disponibles dans la section Référence de ce site. Utilisez ces pages pour vous assurer que votre site Starlight est configuré et fonctionne correctement.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/getting-started.mdx:91]:
```suggestion
Les informations de [configuration de projet](/fr/reference/configuration/) et [configuration du frontmatter de page individuelle](/fr/reference/frontmatter/) pour Starlight sont disponibles dans la section Référence de ce site. Utilisez ces pages pour vous assurer que votre site Starlight est configuré et fonctionne correctement.
```

File: docs/src/content/docs/fr/getting-started.mdx [hunk 2]
```diff
@@ -85,3 +85,13 @@ Si vous oubliez quelque chose d'important, Starlight vous le fera savoir.
...
+
+Voyez la liste croissante de guides dans la barre latérale pour vous aider à ajouter du contenu et à personnaliser votre site Starlight.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/getting-started.mdx:93]:
```suggestion
Utilisez la liste croissante de guides dans la barre latérale pour vous aider à ajouter du contenu et à personnaliser votre site Starlight.
```

File: docs/src/content/docs/fr/getting-started.mdx [hunk 3]
```diff
@@ -85,3 +85,13 @@ Si vous oubliez quelque chose d'important, Starlight vous le fera savoir.
...
+
+Si vous ne trouvez pas de réponse dans cette documentation, veuillez consulter la [documentation complète d'Astro](https://docs.astro.build) pour obtenir la documentation complète d'Astro. Votre question peut être résolue en comprenant comment Astro fonctionne en général, sous ce thème Starlight.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/getting-started.mdx:95]:
```suggestion
Si vous ne trouvez pas votre réponse dans cette documentation, veuillez consulter la [documentation complète d'Astro](https://docs.astro.build). Votre question peut être résolue en comprenant comment Astro fonctionne en général, sous ce thème Starlight.
```

File: docs/src/content/docs/fr/getting-started.mdx [hunk 4]
```diff
@@ -85,3 +85,13 @@ Si vous oubliez quelque chose d'important, Starlight vous le fera savoir.
...
+
+Vous pouvez également obtenir de l'aide dans le [Discord Astro](https://astro.build/chat/) de notre communauté active et sympathique ! Vous pouvez poster des questions dans notre forum `#support` ou visiter notre canal dédié `#starlight` pour discuter du développement en cours et plus encore !
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/getting-started.mdx:97]:
```suggestion
Vous pouvez également obtenir de l'aide dans le [Discord Astro](https://astro.build/chat/) de notre communauté active et sympathique ! Vous pouvez poster des questions dans notre forum `#support` ou visiter notre canal dédié `#starlight` pour discuter des développements en cours et plus encore !
```

**Review by HiDeoo — COMMENTED** [review]
Neat, I left a few suggestions to avoid some repetitions.

**Comment by dreyfus92** [comment]
Thank you so much for reviewing my PR @HiDeoo, all suggestions were applied 😁

File: docs/src/content/docs/fr/getting-started.mdx [hunk 5]
```diff
@@ -85,3 +85,13 @@ Si vous oubliez quelque chose d'important, Starlight vous le fera savoir.
...
 
 [Pour en savoir plus sur le déploiement d'un site Astro, consultez la documentation Astro](https://docs.astro.build/fr/guides/deploy/)
+
+## Dépannage de Starlight
+
+Les informations [configuration de projet](/fr/reference/configuration/) et [configuration du frontmatter de la page individuelle](/fr/reference/frontmatter/) pour Starlight sont disponibles dans la section Référence de ce site. Utilisez ces pages pour vous assurer que votre site Starlight est configuré et fonctionne correctement.
+
+Utilisez la liste croissante de guides dans la barre latérale pour vous aider à ajouter du contenu et à personnaliser votre site Starlight.
+
+Si vous ne trouvez pas votre réponse dans cette documentation, veuillez consulter la [documentation complète d'Astro](https://docs.astro.build) pour obtenir la documentation complète d'Astro. Votre question peut être résolue en comprenant comment Astro fonctionne en général, sous ce thème Starlight.
```
Comment (delucis) [inline docs/src/content/docs/fr/getting-started.mdx:95]: Looks like part of @HiDeoo’s [suggestion](https://github.com/withastro/starlight/pull/377#discussion_r1271057939) didn’t make it here:

```suggestion
Si vous ne trouvez pas votre réponse dans cette documentation, veuillez consulter la [documentation complète d'Astro](https://docs.astro.build/). Votre question peut être résolue en comprenant comment Astro fonctionne en général, sous ce thème Starlight.
```

**Review by delucis — COMMENTED** [review]
Thanks for translating @dreyfus92 and for reviewing @HiDeoo! Noticed one detail:

**Review by delucis — APPROVED** [review]
Merci @dreyfus92 🙌

**Pull request merged**